lambeq/text2diagram/spiders_reader.py

```diff
@@ -46,3 +48,4 @@ def sentence2diagram(self,
 
 spiders_reader = SpidersReader()
 bag_of_words_reader = spiders_reader
+spider_chain_reader = LinearReader(Spider(AtomicType.SENTENCE, 2, 1))
```

Comment:
`LinearReader` requires a `Diagram` but `Spider` is a `Box`. Should LinearReader be accepting `Box` as well or we should convert the Spider into a diagram, e.g. using `.to_diagram()`?

Reply:
Just use `.to_diagram()` for now. Please also note that your branch requires some cleaning.